action/index.js

```diff
@@ -86,7 +86,10 @@ async function main() {
   });
   core.debug(`Discovered ${assets.size} assets`);
 
-  const api = new API(`GitHubOIDC ${token}`, ORIGIN);
+  const api = new API(`GitHubOIDC ${token}`, ORIGIN, {
+    alwaysPrintDenoRay: true,
+    logger: core,
```

Comment:
I wonder if this need to be an option. How about just importing `"@actions/core"` in `src/utils/api.ts` and calling `core.notice()`?

Reply:
`src/utils/api.ts` is called from not only GitHub Actions implementation but also deployctl (CLI) implementation. I wanted to keep `api.ts` agnostic about these differences.

Reply:
Oh, makes sense